docs/features/resource-reservation.md

```diff
@@ -0,0 +1,231 @@
+# Resource Reservation User Documentation
+
+This document uses a kind cluster as an example to introduce how to enable and use the resource reservation capability of the Godel Scheduler.
+
+## Prerequisites
+
+Please make sure that the Godel Controller Manager component has been deployed in the kind cluster along with other scheduler components as control plane components.
+
+```shell
+$ k get po -n godel-system
+NAME                                  READY   STATUS    RESTARTS   AGE
+binder-68dc8bdf6b-v7m59               1/1     Running   0          2d20h
+controller-manager-5b8fcf5f48-mh2bg   1/1     Running   0          2d20h
+dispatcher-5bd97db98f-vw9rv           1/1     Running   0          2d20h
+scheduler-794f9b4c67-mnlln            1/1     Running   0          2d20h
+```
+
+### controller manager
+
+Controller manager is the control plane component of the Godel Scheduler, responsible for performing the CRUD operations on the Reservation CR.
+
+The optional startup parameter `--reservation-ttl` is used to control the Reservation CR timeout (default value: '60', unit: seconds). If the Reservation CR is not matched after the timeout, the controller manager will delete the Reservation object to notify the scheduler to release the reserved resources.
+
+In addition, the global parameter `--reservation-ttl` can be set manually on the Pod or Deployment object annotation to override the controller manager's global parameter.
+
+```yaml
+godel.bytedance.com/reservation-ttl=7200
+```
+
+For example, if you configure the above annotation on a Pod or Deployment, the scheduler will automatically release the reserved resources after 24 hours if there is no consumer for the reserved resources.
```

Comment:
What if both of the Deployment and Pod configured this field?

Priority needs to be clearly described.

Reply:
added some notes about the priority